src/containerapp/azext_containerapp/_validators.py

```diff
@@ -49,6 +50,13 @@ def validate_create(registry_identity, registry_pass, registry_user, registry_se
         raise InvalidArgumentValueError("--registry-identity: expected an ACR registry (*.azurecr.io) for --registry-server")
 
 
+def validate_runtime(runtime, enable_java_metrics):
+    if runtime and runtime.lower() not in SUPPORTED_RUNTIME_LIST:
+        raise ValidationError(f"Runtime {runtime} is not supported. Supported runtimes are {', '.join(SUPPORTED_RUNTIME_LIST)}.")

```

Comment:
If the param is defined with `arg_type=get_enum_type()`, this validation logic can be removed.